sdk/core/azure-core-v2/src/main/java/com/azure/v2/core/implementation/AccessTokenCache.java

```diff
@@ -160,8 +155,8 @@ private static void logTokenRefresh(LogLevel level, AccessToken cache, OffsetDat
                 cache.getExpiresAt(), tte.abs().getSeconds(), REFRESH_DELAY_STRING, tte.isNegative()));
```

Comment:
this is as close as it will let me leave the comment. Can you rename `checkIfForceRefreshRequired` to `checkIfTokenRequestsAreTheSame` or something? Bonus points for breaking the if statement apart so it's more readable.